WORKING_GROUPS.md

```diff
@@ -0,0 +1,173 @@
+# IPFS Project Working Groups
+
+## Index
+
+- [Description](#description)
+- [Working Group Lifecycle](#working-group-lifecycle)
+- [Current Working Groups](#current-working-groups)
+- [Other](#other)
+
+## Description
+
+The IPFS Project Working Groups are teams of people that are appointed to research, develop and deploy new work under the scope that the working group is focused. This structure is designed to provide clarity and direction to the project, enabling individual contributors to focus their time and energy on the areas they are most interested.
```

Comment:
why specifically mention _new_ work? a lot of these WGs do ongoing work that needs perennial attention.

Reply:
Perhaps _new_ was a poor word choice here. I meant _new_ as in their are creating that in the scope. I'll remove it as it is not necessary.